sys/include/embUnit.h

```diff
@@ -9,9 +9,9 @@
 /**
  * @defgroup unittests Unittests
  * @ingroup  sys
+ * @brief RIOT Unittest Framework
```

Comment:
It's not *the* RIOT unittest framework, its [embUnit](http://embunit.sourceforge.net/embunit/) (maybe add a link to their SourceForge, too). The header here just contains some simplifications for their API.